package.json

```diff
@@ -22,8 +22,11 @@
   "homepage": "https://github.com/northcoders/be-nc-news#readme",
   "devDependencies": {
     "husky": "^8.0.2",
+    "install": "^0.13.0",
```

Comment:
Interesting package installed here called "install"! Did you run `npm install install`? I also can't see any changes to the endpoints.json in this pull request, don't forget to update that with your new endpoints!